content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -1,48 +1,98 @@
 
-.. image:: close_tickets/customer-view-close-ticket.png
-   :alt: Customer view of ticket closing in Odoo Helpdesk.
+Go to the team's settings page by going to :menuselection:`Helpdesk --> Configuration --> Teams`.
+Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.
+
+.. image:: close_tickets/closing_automatic_setting.png
+   :align: center
+   :alt: Automatic ticket closing setting options on team page
+
+If one of the team's stages is set to be folded in the kanban view, it will be the default selection
+in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the stage that
+occurs first in the pipeline will be the default. If no stage is set to be folded, the default
+selection will be the last stage in the pipeline.
+
+Adjust the :guilabel:`After days of inactivity` field to the appropriate number of days.
+
+.. important::
+   The :guilabel:`After days of inactivity` field does not take the working calendar into account
+   when tracking time.
+
+If only certain stages should be used to track days of inactivity, they can be added in the
+:guilabel:`In Stages` field.
+
+.. example::
+   A pipeline has the following stages:
+
+   - New
+   - In Progress
+   - Customer Feedback
+   - Closed
+
+   Tickets may linger in the *Customer Feedback* stage, because once an issue is solved, customers do
+   not respond right away. At that point, they can be closed automatically. However, tickets in the
+   *New* and *In Progress* stages may remain inactive due to assignment or workload issues. Closing
+   them automatically may result in issues going unsolved.
 
-Get reports on tickets closed by customers
-==========================================
+   Therefore, the :guilabel:`Automatic Closing` settings would be configured as below-
 
-To analyze the tickets that have been closed by customers, go to :menuselection:`Helpdesk -->
-Reporting --> Tickets`. Then, click on the :guilabel:`Filters` menu and choose :guilabel:`Add
-Custom filter`. Next, set the custom filter parameters to :guilabel:`Closed by partner` and
-:guilabel:`is true`. Finally, click :guilabel:`Apply`.
+   .. image:: close_tickets/closing_example.png
+      :align: center
+      :alt: Example Automatic Closing settings
 
-.. image:: close_tickets/closed-by-search-filter.png
-   :alt: Filter for tickets closed by customers on Odoo Helpdesk's reporting page.
+
+Closure by customers
+====================
+
+The :guilabel:`Closure by Customers` setting allows customers to close their own tickets when they
+determine the issue is resolved.
+
+Start by navigating to :menuselection:`Helpdesk --> Configuration --> Teams` and choose
+the appropriate team. Scroll to the :guilabel:`Self-Service` section and check the box for
+:guilabel:`Closure by Customers`.
+
+.. image:: close_tickets/closing_bycustomer_setting.png
+   :align: center
+   :alt: Customer closing setting in Odoo Helpdesk.
+
+.. note::
```

Comment:
Re: this .. note:: -- it's pretty similar to the summary of folded stages in lines 41-44. Is it worth reiterating in this note, or is it redundant? Maybe the note could be moved up there? Totally up to you!

Reply:
I agree, it is redundant, and can be removed 👍